cmds/more/more.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// More pages through files without any terminal trickery.
+//
+// Synopsis:
+//     more [OPTIONS] FILE
+//
+// Options:
+//     --lines NUMBER: screen size in number of lines
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+
+	flag "github.com/spf13/pflag"
+)
+
+var lines = flag.Int("lines", 40, "screen size in number of lines")
+
+func main() {
+	flag.Parse()
+	if flag.NArg() != 1 {
```

Comment:
Why not just make f os.Stdin in this case?

Reply:
We're also reading from stdin down below, blocking on the user pressing return. They will conflict and the input would page itself with its own newlines.